lib/chef/resource/windows_user_privilege.rb

```diff
@@ -136,11 +136,7 @@ class WindowsUserPrivilege < Chef::Resource
         description: "Privilege to set for users.",
         required: true,
         coerce: proc { |v| v.is_a?(String) ? Array[v] : v },
-        callbacks: {
-           "Option privilege must include any of the: #{privilege_opts}" => lambda { |v|
-             (privilege_opts & v).size == v.size
-           },
-         }
+        callbacks: { "Option privilege must include any of the: #{privilege_opts}" => proc { |n| (Array(n) - privilege_opts).empty? } }
```

Comment:
This would be a reall good thing to test with a unit test since you can make sure it doesn't raise the validation error when you pass in an array.

Reply:
this whole resource needs unit tests.  hopefully I can make some time for that next week, but unfortunately won't be able to add them in right now.